### PR TITLE
Lifecycle cleanup

### PR DESCRIPTION
**Pull request by rlespinasse**

### CODE REVIEW

File: steps/50-lifecycle-lab1-marketplace-solution/.github/workflows/release-workflow.yml [hunk 1]
```diff
@@ -9,6 +9,7 @@ jobs:
 
   release:
     runs-on: ubuntu-latest
+    if: github.ref == 'refs/heads/master'
```
Comment (ThibDujardin) [inline steps/50-lifecycle-lab1-marketplace-solution/.github/workflows/release-workflow.yml:12]:
```suggestion
    if: github.ref == 'refs/heads/main'
```
Master is deprecated

File: steps/50-lifecycle-lab2-dist-solution/.github/workflows/release-workflow.yml [hunk 2]
```diff
@@ -13,6 +13,7 @@ jobs:
 
   release:
     runs-on: ubuntu-latest
+    if: github.ref == 'refs/heads/master'
```
Comment (ThibDujardin) [inline steps/50-lifecycle-lab2-dist-solution/.github/workflows/release-workflow.yml:16]:
```suggestion
    if: github.ref == 'refs/heads/main'
```
Master is deprecated

File: steps/50-lifecycle-lab2-dist/.github/workflows/release-workflow.yml [hunk 3]
```diff
@@ -11,6 +11,7 @@ jobs:
 
   release:
     runs-on: ubuntu-latest
+    if: github.ref == 'refs/heads/master'
```
Comment (ThibDujardin) [inline steps/50-lifecycle-lab2-dist/.github/workflows/release-workflow.yml:14]:
```suggestion
    if: github.ref == 'refs/heads/main'
```
Master is deprecated

**Review by ThibDujardin — CHANGES_REQUESTED** [review]

File: steps/50-lifecycle-lab1-marketplace-solution/.github/workflows/release-workflow.yml (hunk 1 above)
Comment (rlespinasse) [reply steps/50-lifecycle-lab1-marketplace-solution/.github/workflows/release-workflow.yml:12]: But not in semantic release 😄 (I have add a warning on this during the lab).

**Review by ThibDujardin — APPROVED** [review]

**Pull request merged**